code/game/objects/items/storage/surgical_tray.dm

```diff
@@ -3,7 +3,7 @@
 	desc = "A small metallic tray covered in sterile tarp. Intended to store surgical tools in a neat and clean fashion."
 	icon_state = "surgical_tray"
 	flags_atom = CONDUCT
-	w_class = WEIGHT_CLASS_BULKY //Should not fit in backpacks
+	w_class = WEIGHT_CLASS_NORMAL //Should not fit in vests or pouches
```

Comment:
Leave the tray as is. Instead, add the surgical vest to the medic vendor and add an additional slot. That way it can fit the whole array of surgical tools.

Reply:
done